src/chrome/content/rules/Private-Layer.com.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="Private Layer" platform="cacert" default_off="missing certificate chain">
```

Comment:
is this still cacert? [not a huge fan of adding default_off rulesets in general]